src/main/java/ch/njol/skript/expressions/ExprNoDamageTicks.java

```diff
@@ -12,66 +18,88 @@
 import ch.njol.skript.expressions.base.SimplePropertyExpression;
 import ch.njol.util.coll.CollectionUtils;
 
+import java.util.function.Consumer;
+
 @Name("No Damage Ticks")
 @Description("The number of ticks that an entity is invulnerable to damage for.")
-@Examples({"on damage:",
-		"	set victim's invulnerability ticks to 20 #Victim will not take damage for the next second"})
-@Since("2.5")
-public class ExprNoDamageTicks extends SimplePropertyExpression<LivingEntity, Long> {
+@Examples({
+	"on damage:",
+		"\tset victim's invulnerability ticks to 20 #Victim will not take damage for the next second",
+	"",
+	"if the no damage timespan of {_entity} is 0 seconds:",
+		"\tset the invincibility time of {_entity} to 1 minute"
+})
+@Since("2.5, INSERT VERSION (timespan)")
+public class ExprNoDamageTicks extends SimplePropertyExpression<LivingEntity, Object> {
 	
 	static {
-		register(ExprNoDamageTicks.class, Long.class, "(invulnerability|invincibility|no damage) tick[s]", "livingentities");
+		registerDefault(ExprNoDamageTicks.class, Object.class,"(invulnerability|invincibility|no damage) tick[s]", "livingentities");
+		registerDefault(ExprNoDamageTicks.class, Object.class, "(timespan:(invulnerability|invincibility|no damage) time[[ ]span])", "livingentities");
```

Comment:
Why two registrations?
The problem here is then it doubles up the docs.
You could just do a single registration with an option of ticks or timespan
(Hopefully I typed this change out correctly)
```suggestion
		registerDefault(ExprNoDamageTicks.class, Object.class, "(invulnerability|invincibility|no damage) (timespan:time[ ]span|tick[s])", "livingentities");
```